### PR TITLE
pkg-config: add page

### PR DESCRIPTION
**Pull request by hojjatabdollahi**
- [x] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).

### CODE REVIEW

File: pages/linux/pkg-config.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# pkg-config
+
+> Return metainformation about installed libraries.
+
+- Get the list of libraries and their dependencies:
+
+`pkg-config --libs {{library1 library2 ...}}`
+
+- Get the list of libraries, their dependencies, and proper cflags for using with gcc:
```
Comment (zdroid) [inline pages/linux/pkg-config.md:9]:
```suggestion
- Get the list of libraries, their dependencies, and proper cflags for gcc:
```

File: pages/linux/pkg-config.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
+# pkg-config
+
+> Return metainformation about installed libraries.
```
Comment (andrik) [inline pages/linux/pkg-config.md:3]:
```suggestion
> Tool to provide the details of installed libraries for compiling applications
```

Comment (zdroid) [reply pages/linux/pkg-config.md:3]: @andrik Maybe this is even better?
```suggestion
> Provide the details of installed libraries for compiling applications.
```

Comment (andrik) [reply pages/linux/pkg-config.md:3]: @zdroid I updated your suggestion adding `More information` section

**Comment by hojjatabdollahi** [comment]
Hi @andrik and @zdroid,
Thanks for your comments. I agree with both of you. My original description was from the man pages. But this description is better, especially for the beginners.

**Comment by hojjatabdollahi** [comment]
@andrik , @zdroid 
Originally I wanted to add an example with a compiler as well, but there was some error. Do you think I should try and add it one more time?

- Compile your code with `libwebkit2gtk-4.0`:
``c++ main.c `pkg-config --cflags --libs webkit2gtk-4.0` ``

**Comment by sbrl** [comment]
Hey, @hojjatabdollahi! Thanks for the new page.

That example looks good. What do you mean by error? If you mean when trying to edit, try using the GitHub web interface.

Also, perhaps we could adjust that example there slightly. What about this?
```markdown
- Compile your code with libwebkit2gtk-4.0:

`c++ main.c "$(pkg-config --cflags --libs webkit2gtk-4.0)"`
```

The `$()` syntax is newer and more robust than the older backtick syntax.

**Comment by hojjatabdollahi** [comment]
@sbrl 
Thank you. This might solve the "check error", I had to use double backticks and mix them with single backticks and I think the check failed because of that.
However, the double quotations in your example will result in an error, but the same works without the double qoutations. Should I go ahead and add it to the page?
- Compile your code with libwebkit2gtk-4.0:

`c++ main.c "$(pkg-config --cflags --libs webkit2gtk-4.0)"`

**Comment by sbrl** [comment]
Go ahead, @hojjatabdollahi :smiley_cat:

**Pull request merged**